src/libraries/System.Runtime/tests/System/TimeZoneInfoTests.cs

```diff
@@ -2321,6 +2321,46 @@ public static void TestNameWithInvariantCulture()
 
         }
 
+        [Theory]
+        [PlatformSpecific(TestPlatforms.AnyUnix)]
```

Comment:
Is this test specific to wasm? IIUC, the `backwards` file is being added only for that? Then maybe we should use `[PlatformSpecific(TestPlatforms.Browser)]`?

Reply:
This isn't really specific to browser although it would be unlikely to fail anywhere else.